[2023-06-26T06:31:24.379+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: execute_python.check_participants scheduled__2023-06-09T22:00:00+00:00 [queued]>
[2023-06-26T06:31:24.385+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: execute_python.check_participants scheduled__2023-06-09T22:00:00+00:00 [queued]>
[2023-06-26T06:31:24.386+0200] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-06-26T06:31:24.405+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_participants> on 2023-06-09 22:00:00+00:00
[2023-06-26T06:31:24.409+0200] {standard_task_runner.py:57} INFO - Started process 100639 to run task
[2023-06-26T06:31:24.412+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'execute_python', 'check_participants', 'scheduled__2023-06-09T22:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/thesis_container.py', '--cfg-path', '/tmp/tmptdga2akf']
[2023-06-26T06:31:24.413+0200] {standard_task_runner.py:85} INFO - Job 89: Subtask check_participants
[2023-06-26T06:31:24.514+0200] {task_command.py:410} INFO - Running <TaskInstance: execute_python.check_participants scheduled__2023-06-09T22:00:00+00:00 [running]> on host DESKTOP-3KJU1M7.
[2023-06-26T06:31:24.592+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='execute_python' AIRFLOW_CTX_TASK_ID='check_participants' AIRFLOW_CTX_EXECUTION_DATE='2023-06-09T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-09T22:00:00+00:00'
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - ['reynel_95']
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - []
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - ['reynel_95']
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - New participants found. Creating containers...
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - 0
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - <class 'int'>
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - ['reynel_95']
[2023-06-26T06:31:24.594+0200] {logging_mixin.py:149} INFO - list test ['reynel_95']
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/Participant1/1_6-6.mrxs']
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - single container reynel_95
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/Participant1/1_6-6.mrxs
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/Participant1/1_6-6.mrxs
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/Participant1
[2023-06-26T06:31:24.595+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/Participant1/1_6-6.mrxs']
[2023-06-26T06:31:25.050+0200] {logging_mixin.py:149} INFO - Container 1_6-6.mrxs image built successfully.
[2023-06-26T06:31:26.676+0200] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 263, in check_participants
    build_and_run_containers(index)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 121, in build_and_run_containers
    subprocess.run(run_command, shell=True, check=True)
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'docker run -d --name 1_6-6.mrxs 1_6-6.mrxs' returned non-zero exit status 125.
[2023-06-26T06:31:26.681+0200] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=execute_python, task_id=check_participants, execution_date=20230609T220000, start_date=20230626T043124, end_date=20230626T043126
[2023-06-26T06:31:26.692+0200] {standard_task_runner.py:104} ERROR - Failed to execute job 89 for task check_participants (Command 'docker run -d --name 1_6-6.mrxs 1_6-6.mrxs' returned non-zero exit status 125.; 100639)
[2023-06-26T06:31:26.719+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-26T06:31:26.749+0200] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
